contracts/OBSSStorage.sol

```diff
@@ -198,6 +198,10 @@ contract OBSSStorage is OwnableUpgradeable, ERC2771Recipient {
   // KetlCred
 
   function grantKetlCred(AddReactionRequest memory reactionRequest) internal {
+    Post memory post = feeds.getPost(
```

Comment:
I wonder if we do this in `feed.addReaction` too? Maybe we should pass the `post` to `feed.addReaction`?

Reply:
We don't fetch posts anywhere else. I did think about moving this to `addFeedReaction` and only calling `grantKetlCred` if the reaction was an upvote, but that logic would be repeated in `addProfileReaction`, so I opted to keep the logic in `grantKetlCred` instead.